added mapping for members, along with their information and images

tarillo opened the pull request:
I don't know how to get it to appear so I couldn't take screenshot of appearance.
Having trouble with image appearance in MemberCard.tsx as well

agoel27 commented:
In order to get it to appear, follow the following steps:
1. create a folder called data in the src folder
2. move the data file (members.ts) to this data folder
3. update the import in Members.tsx to `import { members } from "@/data/members";`
4. create a folder called Board in the app folder
5. create a file called page.tsx in the Board folder
6. create a component called Board with the Members component something like this:

```
import Members from "@/components/Board/Members";

const Board = () => {
  return (
    <>
      <Members />
    </>
  );
};

export default Board;
```

Also, make sure to use @ for imports instead of ./
@ is an alias for the src folder

tarillo commented:
![Screenshot 2025-05-12 163451](https://github.com/user-attachments/assets/d8e4c8c2-19f8-4482-b3fa-9313e4897bdb)

-Was able to get member cards to display

-Fixed the MemberCards appearance as it looked off

-Added padding around the cards as well as they were intersecting with nav bar and footer

agoel27 commented on src/app/board/page.tsx:
```diff
@@ -0,0 +1,11 @@
+
+const Board = () => {
+  return (
+    <div className="px-10 py-10">
```
```suggestion
    <div className="px-10 py-10 bg-3d-orange-secondary">
```

agoel27 commented on src/components/Board/MemberCard.tsx:
```diff
@@ -1,25 +1,36 @@
             <div className="bg-3d-blue-primary font-francois items-center justify-center rounded-lg">
-              <p className="text-center text-2xl text-black">Timothy Lee</p>
-              <p className="text-center text-2xl text-black">President</p>
+              <p className="text-center text-2xl text-black">{name}</p>
```
```suggestion
              <p className="text-center text-2xl text-black whitespace-nowrap">{name}</p>
```

agoel27 commented on src/components/Board/MemberCard.tsx:
```diff
@@ -1,25 +1,36 @@
-              <p className="text-center text-2xl text-black">Timothy Lee</p>
-              <p className="text-center text-2xl text-black">President</p>
+              <p className="text-center text-2xl text-black">{name}</p>
+              <p className="text-center text-2xl text-black">{title}</p>
```
```suggestion
              <p className="text-center text-2xl text-black whitespace-nowrap">{title}</p>
```

agoel27 commented on src/components/Board/MemberCard.tsx:
```diff
@@ -1,25 +1,36 @@
-      <div className="bg-3d-red-primary relative -bottom-5 flex h-80 w-60 items-center justify-center rounded-lg p-2.5">
+      <div className="bg-3d-red-primary relative -bottom-5 flex h-80 w-60 items-center justify-center rounded-lg p-2">
         <div className="h-3/4 w-full rounded-xs border-4 border-white p-2">
           <div className="h-full w-full items-center justify-center border-gray-300 bg-gray-200">
```
```suggestion
          <div className="h-full w-full items-center justify-center">
```

agoel27 commented on src/components/Board/MemberCard.tsx:
```diff
@@ -1,25 +1,36 @@
       boxPadding="pr-0.2 pl-0.2 py-0"
     >
-      <div className="bg-3d-red-primary relative -bottom-5 flex h-80 w-60 items-center justify-center rounded-lg p-2.5">
+      <div className="bg-3d-red-primary relative -bottom-5 flex h-80 w-60 items-center justify-center rounded-lg p-2">
```
instead of using large height and width values, try using a % value or vh values

agoel27 commented on src/components/Board/MemberCard.tsx:
```diff
@@ -1,25 +1,36 @@
     <BoxShadow
       boxColor="bg-3d-red-primary"
       boxShadow="shadow-3d-red-secondary rounded-sm"
       boxPadding="pr-0.2 pl-0.2 py-0"
```
```suggestion
```
